tests/server/blueprints/cases/test_matchmaker_controllers.py

```diff
@@ -107,14 +106,19 @@ def test_matchmaker_add_snv(app, user_obj, case_obj, test_hpo_terms, mocker):
         )
         assert saved_results == 1
 
-        # THEN ine case should be uodated and a submission object correctly saved to database
+        # THEN one case should be uodated and a submission object correctly saved to database
         updated_case = store.case_collection.find_one()
         submission = updated_case["mme_submission"]
         assert (
             submission["patients"][0]["genomicFeatures"][0]["variant"]["start"]
             == test_variant["position"]
         )
 
+        # AND the relative event should be saved in the database:
+        mme_event = store.event_collection.find_one()
+        assert mme_event["verb"] == "mme_add"
+        assert mme_event["link"] == f"/{case_obj['owner']}/{case_obj['display_name']}"
```

Comment:
ahem, in this case I guess it's ok if the different context would make url_for() return something slightly different.

Reply:
I think url_for is more meant to be used on the fly when you load a page. For stuff that that is saved to the database in all events it is just saved like this. It's either case links (`/instxxx/case_name`) or variant links (`/instxxx/case_name/variant_name`)

Reply:
Ok, you made your point, url_for creates the same links 😄 Switching to url_for now!